contracts/common/SecuredTokenTransfer.sol

```diff
@@ -1,13 +1,19 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 pragma solidity >=0.7.0 <0.9.0;
 
-/// @title SecuredTokenTransfer - Secure token transfer
-/// @author Richard Meissner - <richard@gnosis.pm>
+/**
+ * @title SecuredTokenTransfer - Secure token transfer.
+ * @author Richard Meissner - @rmeissner
+ */
 contract SecuredTokenTransfer {
-    /// @dev Transfers a token and returns if it was a success
-    /// @param token Token that should be transferred
-    /// @param receiver Receiver to whom the token should be transferred
-    /// @param amount The amount of tokens that should be transferred
+    /**
+     * @notice Transfers token and returns a boolean if it was a success
```

Comment:
`Transfers a token and returns a boolean if it was a success`, why are you removing `a`?

Reply:
Idk what happened here